Twine on Travis is getting auth failure

Need to print third-party process logs in case of failure for upload and other tasks.

related to #531

```
BUILD FAILED - Error while executing Twine ['upload', '/home/travis/build/pybuilder/pybuilder/target/dist/pybuilder-0.12.0.dev/dist/pybuilder-0.12.0.dev20171123180925.tar.gz', '/home/travis/build/pybuilder/pybuilder/target/dist/pybuilder-0.12.0.dev/dist/pybuilder-0.12.0.dev20171123180925-py2-none-any.whl']. See /home/travis/build/pybuilder/pybuilder/target/reports/distutils/twine_upload.log for full details:
	  0%|          | 0.00/128K [00:00<?, ?bytes/s]
	  6%|6         | 8.19K/128K [00:00<00:02, 58.8Kbytes/s]
	                                                       
	Uploading distributions to https://upload.pypi.org/legacy/
	Uploading pybuilder-0.12.0.dev20171123180925-py2-none-any.whl
	HTTPError: 403 Client Error: Invalid or non-existent authentication information. for url: https://upload.pypi.org/legacy/
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -21,7 +21,7 @@ env:
   - DEPLOY_BRANCHES: "'master 0.11'"
   - DEPLOY_PYTHONS: "'2.7 3.6'"
   - PYB_ARGS: "'-v -X install_dependencies analyze install'"
-  - TWINE_USERNAME: pybuilder_travis2
+  - TWINE_USERNAME: pybuilder-travis2
   - secure: D49QJr9Z4v9yz9VObFw0q44o2fy3YBjI48b6+Vb1qr2RNMF5GpVj+xWFTr9j1qMw7ilfwWCEKlHytKmh/OSOStUcd1vTlasGz8YefBCP8iIbvp9M6qcBKqMlp8f/6uxwUxak+OMvj5WuKwtyIUy8L9JwDxd02Q12MUW6SNtu2Lo=
 before_script:
 - pip install coveralls

